### PR TITLE
Feature/define controller type

### PR DESCRIPTION
**Pull request by ahmedggalal**
add the ability to create modules using :
module:make User --web
module:make User --api
module:make User --plain

### CODE REVIEW

File: src/Commands/ModuleMakeCommand.php [hunk 1]
```diff
@@ -68,8 +69,14 @@ protected function getOptions()
...
             ['plain', 'p', InputOption::VALUE_NONE, 'Generate a plain module (without some resources).'],
+            ['api', null, InputOption::VALUE_NONE, 'Generate an api module.'],
+            ['web', null, InputOption::VALUE_NONE, 'Generate a web module.'],
             ['disabled', 'd', InputOption::VALUE_NONE, 'Do not enable the module at creation.'],
             ['force', null, InputOption::VALUE_NONE, 'Force the operation to run when the module already exists.'],
         ];
     }
+
+    private function getModuleType($isPlain,$isApi,$isWeb){
+        return ($isPlain && $isApi && $isWeb)||($isPlain && $isApi)||($isApi && $isWeb)||($isPlain && $isWeb)?'web':($isPlain&&(!$isApi&&!$isWeb)?'plain':($isApi&&(!$isPlain&&!$isWeb)?'api':'web'));
```
Comment (nWidart) [inline src/Commands/ModuleMakeCommand.php:80]: I have no clue wtf this does.

Comment (ahmedggalal) [reply src/Commands/ModuleMakeCommand.php]: it just returns web if users entered more that one options , --plain --web --api

Comment (nWidart) [reply src/Commands/ModuleMakeCommand.php]: yes I guessed that, but it's unreadable. The new version is still unreadable. Don't use ternaries here.
Also please create tests for each possible outcome.

**Comment by nWidart** [comment]
Thank you for the changes! 👍

**Comment by ahmedggalal** [comment]
#947

**Pull request merged**